crates/turborepo-auth/src/auth/login.rs

```diff
@@ -20,13 +30,35 @@ pub async fn login<'a>(
     existing_token: Option<&'a str>,
     login_url_configuration: &str,
     login_server: &impl LoginServer,
-) -> Result<Cow<'a, str>, Error> {
-    // Check if token exists first.
+) -> Result<Token, Error> {
+    // Check if passed in token exists first.
     if let Some(token) = existing_token {
         if let Ok(response) = api_client.get_user(token).await {
             println!("{}", ui.apply(BOLD.apply_to("Existing token found!")));
             ui::print_cli_authorized(&response.user.email, ui);
-            return Ok(token.into());
+            return Ok(Token {
+                token: token.into(),
+                is_existing: true,
+            });
+        }
+    }
+
+    // If the user is logging into Vercel, check for an existing `vc` token.
+    if login_url_configuration.contains("vercel.com") {
+        match extract_vercel_token(api_client).await {
+            Ok(token) => {
+                println!(
+                    "{}",
+                    ui.apply(BOLD.apply_to("Existing Vercel token found!"))
+                );
+                return Ok(token);
+            }
+            Err(error) => {
+                // Only send the warning if we're debugging.
+                if cfg!(debug_assertions) {
+                    warn!("Failed to extract Vercel token: {error}");
+                }
```

Comment:
Do you want this to be emitted when `-vv` is passed or just on the debug build of `turbo`?

Also with the current setup this error can come from either us failing to find the vercel config dir or an issue hitting the API.

Reply:
I made the suggestions to the extract logic so that now it can only be if we didn't find the config dir. I was waffling about whether we want this just for the debug build or via flags, since this shouldn't stop us from logging in, and I didn't think we needed an extra log line telling the user we didn't find the config dir, but I'm open to thoughts here

Reply:
Just putting this as a debug log makes more sense to me? It allows us to see if a user is having issues with a vercel token without having to give them a special build.